UserDetailsClient/UserDetailsClient.Core/MainPage.xaml.cs

```diff
@@ -14,6 +14,33 @@ public partial class MainPage : ContentPage
         public MainPage()
         {
             InitializeComponent();
+            CheckForAccountsAsync();
```

Comment:
Doesn't this (calling AcquireTokenSilent from the constructor) provoke the application to look blank for a while (until the OIDC metadata document is read) ?
Wouldn't it be better to cal this when the application is realized?